tests/Controller/Customer/RegisterApiTest.php

```diff
@@ -55,16 +60,19 @@ public function it_allows_to_register_in_shop_and_sends_a_verification_email_if_
         Assert::assertNotNull($user);
         Assert::assertFalse($user->isEnabled());
 
-        /** @var EmailCheckerInterface $emailChecker */
-        $emailChecker = $this->get('sylius.behat.email_checker');
-        Assert::assertTrue($emailChecker->hasRecipient('vinny@fandf.com'));
+        $email = self::getMailerMessage();
+        self::assertEmailAddressContains($email, 'to', 'vinny@fandf.com');
```

Comment:
Can it be done that way?
```suggestion
        if (!self::isSymfonyMailerAvailable()) {
            /** @var EmailCheckerInterface $emailChecker */
            $emailChecker = $this->get('sylius.behat.email_checker');
            Assert::assertTrue($emailChecker->hasRecipient('vinny@fandf.com'));
        } else {
            $email = self::getMailerMessage();
            self::assertEmailAddressContains($email, 'to', 'vinny@fandf.com');
        }
```
Thanks to this we don't need to skip the test.